src/cloud/digitalocean/provisioner.ts

```diff
@@ -127,17 +170,9 @@ class Provisioner {
           });
         }
       }
-      return Promise.reject({
-        'errcode': 'VM_DNE',
-        'message': 'Droplet ' + name + ' doesnt exist'
+      return Promise.resolve({
```

Comment:
Okay, I updated `getDroplet_`. Do you think this is better? Problem is that if `getDroplet_` fails then `start` should continue but if it returns a droplet then `start` should throw an error. So `start` wasn't really simplified, but I do think it's better if `getDroplet_` throws an error.